activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Fix ActiveRecord query attribute method when given Date data-type
```

Comment:
Can you update the "when given Date data-type" part?
Since "when given Date data-type" is a part of what we fixes.